cabot/__init__.py

```diff
@@ -2,4 +2,4 @@
 
 # This will make sure the app is always imported when
 # Django starts so that shared_task will use this app.
-from .celery import app as celery_app
+from .celery import app as celery_app # noqa
```

Comment:
nit: 2 spaces before comment (this actually causes another flake8 error...
`02:02:46 ./cabot/__init__.py:5:38: E261 at least two spaces before inline comment
`